solafune_tools/competition_tools/README.md

```diff
@@ -0,0 +1,67 @@
+# Competitions Tools
+
+## Descriptions
+
+This repository contains a list of competition tools to help solafune's hacker submit their submission and other things related to solafune's competition.
```

Comment:
@fulankun1412 "solafune's hacker" => "Solafune hackers" looks nicer

Reply:
ah and "solafune's competition" => "Solafune competitions"?

Reply:
@shimokuracg So the Solafune competition does not show competition ownership to the Solafune?

Reply:
@fulankun1412 as long as the competitions are operated on our platform, i guess we can call them "Solafune competitions". but are you considering some scenarios?
FYI there are many hosts organizing competitions on Kaggle, and they are generally called "Kaggle Competitions" from users i guess.